nncf/common/utils/os.py

```diff
@@ -14,7 +14,7 @@
 from contextlib import contextmanager
 from pathlib import Path
 
-
+# pylint: disable=W1514
```

Comment:
why not using encoding='utf8'?

Reply:
I thought if somebody wants to open a file in the raw format, which is possible since this function accepts a variable number of arguments, it can lead to an error, if we specify the encoding.